bin/s2nc.c

```diff
@@ -62,17 +61,17 @@ void usage()
     fprintf(stderr, "\n");
     fprintf(stderr, "  -s,--status\n");
     fprintf(stderr, "    Request the OCSP status of the remote server certificate\n");
-    fprintf(stderr, "  --mfl\n");
+    fprintf(stderr, "  -m,--mfl\n");
     fprintf(stderr, "    Request maximum fragment length from: 512, 1024, 2048, 4096\n");
     fprintf(stderr, "  -f,--ca-file [file path]\n");
     fprintf(stderr, "    Location of trust store CA file (PEM format). If neither -f or -d are specified. System defaults will be used.\n");
     fprintf(stderr, "  -d,--ca-dir [directory path]\n");
     fprintf(stderr, "    Directory containing hashed trusted certs. If neither -f or -d are specified. System defaults will be used.\n");
     fprintf(stderr, "  -i,--insecure\n");
     fprintf(stderr, "    Turns off certification validation altogether.\n");
-    fprintf(stderr, "  --cert [file path]\n");
+    fprintf(stderr, "  -l,--cert [file path]\n");
     fprintf(stderr, "    Path to a PEM encoded certificate. Optional. Will only be used for client auth\n");
```

Comment:
What "l"?

Reply:
It was already set in the options struct [here](https://github.com/aws/s2n-tls/blob/589ed7215ef3e36aab4d526401dcee3c2fba91ae/bin/s2nc.c#L274) and in the [switch](https://github.com/aws/s2n-tls/blob/589ed7215ef3e36aab4d526401dcee3c2fba91ae/bin/s2nc.c#L337) statement, just not printed in the help text.